modules/aws/s3.go

```diff
@@ -116,11 +116,11 @@ func CreateS3BucketE(t *testing.T, region string, name string) error {
 	return err
 }
 
-// PutS3BucketVersioningE creates an S3 bucket versioning configuration in the given region against the given bucket name, WITHOUT requiring MFA to remove versioning.
+// PutS3BucketVersioning creates an S3 bucket versioning configuration in the given region against the given bucket name, WITHOUT requiring MFA to remove versioning.
 func PutS3BucketVersioning(t *testing.T, region string, bucketName string) {
 	err := PutS3BucketVersioningE(t, region, bucketName)
 	if err != nil {
-		t.Fatal(err)
+		require.NoError(t, err)
 	}
```

Comment:
Ah actually, the change is to replace the whole if block. As in, this can be:
```go
err := PutS3BucketVersioningE(t, region, bucketName)
require.NoError(t, err)
```

Reply:
Doh, that makes total sense.  I was rushing, sorry!